resources/META-INF/plugin.xml

```diff
@@ -137,11 +137,11 @@
       </ul>
       ]]>
   </change-notes>
-  <version>1.3.6.02112015</version>
+  <version>1.3.6.20151116</version>
   <vendor>Twitter, Inc.</vendor>
 
   <!--if you are changing since-build don't forget to change it in .travis.yml file as well-->
-  <idea-version since-build="143.1" until-build="144.1"/>
+  <idea-version since-build="143.1" until-build="143.381"/>
```

Comment:
Hi Fedor,

Thanks for taking a look. Building pants plugin with 15.0.1 works, but it
has the following error with scala projects for example when importing
/Users/yic/workspace/pants/examples/src/scala/org/pantsbuild/example/hello.
I tried to manually include scala lib 2.10.4 or 2.11.7 in the project sdk
or module sdk, but there is no luck. If you know a way, please do let me
know.

Thanks,
Yi

Cannot find project Scala library 2.10.4 for module
examples_src_scala_org_pantsbuild_example_hello_exe_exe
com.intellij.openapi.externalSystem.model.ExternalSystemException: Cannot
find project Scala library 2.10.4 for module
examples_src_scala_org_pantsbuild_example_hello_exe_exe
at
com.twitter.intellij.pants.service.scala.PantsScalaDataService$$anonfun$4.apply(PantsScalaDataService.scala:59)
at
com.twitter.intellij.pants.service.scala.PantsScalaDataService$$anonfun$4.apply(PantsScalaDataService.scala:59)
at scala.Option.getOrElse(Option.scala:121)
at com.twitter.intellij.pants.service.scala.PantsScalaDataService.com
$twitter$intellij$pants$service$scala$PantsScalaDataService$$doImport(PantsScalaDataService.scala:59)
at
com.twitter.intellij.pants.service.scala.PantsScalaDataService$$anonfun$importData$1.apply(PantsScalaDataService.scala:42)
at
com.twitter.intellij.pants.service.scala.PantsScalaDataService$$anonfun$importData$1.apply(PantsScalaDataService.scala:42)
at scala.collection.immutable.Set$Set1.foreach(Set.scala:79)
at
com.twitter.intellij.pants.service.scala.PantsScalaDataService.importData(PantsScalaDataService.scala:42)
at
com.intellij.openapi.externalSystem.service.project.manage.ProjectDataManager.doImportData(ProjectDataManager.java:254)
at
com.intellij.openapi.externalSystem.service.project.manage.ProjectDataManager.importData(ProjectDataManager.java:151)
at
com.intellij.openapi.externalSystem.service.project.manage.ProjectDataManager.importData(ProjectDataManager.java:206)
at
com.intellij.openapi.externalSystem.service.project.manage.ProjectDataManager.importData(ProjectDataManager.java:212)
at
com.intellij.openapi.externalSystem.util.ExternalSystemUtil$2.onSuccess(ExternalSystemUtil.java:321)
at
com.intellij.openapi.externalSystem.util.ExternalSystemUtil$3.execute(ExternalSystemUtil.java:447)
at
com.intellij.openapi.externalSystem.util.ExternalSystemUtil$4$2.run(ExternalSystemUtil.java:500)
at
com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:563)
at
com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:142)
at
com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:446)
at
com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:392)
at
com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
at
com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:127)
at
com.intellij.openapi.progress.impl.ProgressManagerImpl$1.run(ProgressManagerImpl.java:126)
at
com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:365)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
at java.util.concurrent.FutureTask.run(FutureTask.java:262)
at
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
at
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
at java.lang.Thread.run(Thread.java:745)
at
org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:55)


On Mon, Nov 16, 2015 at 3:11 PM, Fedor Korotkov <notifications@github.com>
wrote:

> In resources/META-INF/plugin.xml
> <https://github.com/pantsbuild/intellij-pants-plugin/pull/70#discussion_r44999471>
> :
>
> >    <vendor>Twitter, Inc.</vendor>
> >
> >    <!--if you are changing since-build don't forget to change it in .travis.yml file as well-->
> > -  <idea-version since-build="143.1" until-build="144.1"/>
> > +  <idea-version since-build="143.1" until-build="143.381"/>
>
> are you sure that you need to change until-build?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pantsbuild/intellij-pants-plugin/pull/70/files#r44999471>
> .
>